### DISCUSSION TITLE
How to close the default `orderBy id`?

### DISCUSSION THREAD

**Discussion by Xishang7707** [prisma/prisma#16630] [Q&A]
![image](https://user-images.githubusercontent.com/38380429/205598950-2bfdf992-4d10-43b5-93e4-912813212914.png)
I set orderby to undefined, but orderby id aes is added by default. how do i remove it?

**Comment by eviefp** [comment]
Hello @Xishang7707. Could you add a few more details to make it easier for us to reproduce this?

Particularly looking for:
- a schema file (preferably minimal to reproduce the problem)
- the query you used that generates the problem above

**Reply by vladi-stevanovic** [reply]
👋 Hi @Xishang7707 , did you perhaps have a chance to check Evie's questions? Let us know if you still need help here!

Or, if you've found a solution, we'd love to know how you solved this problem. Feel free to post it in this thread and "Mark as answer" if this discussion is now solved!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
